index.html

```diff
@@ -28,253 +29,364 @@
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <ul class="work-card__technologies desktop">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                </div>
+                <div class="btn-container">
+                  <button type="button" class="btn">
+                    <a href="#">See Project</a>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </article>
         </li>
       </ul>
     </section>
+    <!-- ABOUT SECTION -->
     <section id="about-container">
-      <ul id="about">
-        <li>
-          <h2 class="header">
-            <span>About<br />Myself</span>
-          </h2>
-        </li>
-        <li>
-          <p class="text">
-            I’m a software developer! I can help you build a product, feature or
-            website. Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contact me.
-          </p>
-        </li>
-        <li>
-          <h2 id="connect">Let's Connect</h2>
-        </li>
-        <li id="social-icons-container">
-          <ul id="social-icons">
-            <li class="icon">
-              <a
-                href="https://github.com/hafedefheij"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <i class="fa-brands fa-github"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a
-                href="https://www.linkedin.com/in/hafedefheij/"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                <i class="fa-brands fa-linkedin-in"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-angellist"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-twitter"></i>
-              </a>
-            </li>
-            <li class="icon">
-              <a href="#" target="_blank" rel="noopener noreferrer">
-                <i class="fa-brands fa-medium"></i>
-              </a>
-            </li>
-          </ul>
-        </li>
-        <li>
-          <div id="button-container">
-            <button type="button" class="btn">
-              <a href="#">Get my resume</a>
-            </button>
+      <article id="about">
+        <div id="description-container">
+          <div id="description">
+            <h2 class="header">
+              <span>About<br />Myself</span>
+            </h2>
```

Comment:
- [ OPTIONAL] I noticed that on the mobile view, this element is not identical to that of the Figma template. When executing responsive designs, style declarations for the desktop view should not affect the mobile view. I would advise you to work on this. Also, notice the color used `#172B4D`

### Your Design

![image](https://user-images.githubusercontent.com/77106962/206790406-dbde5ac0-16b1-4b85-8734-693840ab89ad.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/206790525-869eda57-70dc-49f8-8416-4059fe130129.png)